CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
-# Change Log
-All notable changes to this project will be documented in this file.
-This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
-
-## [Unreleased]
-
-
-## [1.0.0] (2018-05-17)
-### Added
-- *First release* @jmonterrubio
-
-[Unreleased]: https://github.com/idealista/nexus-role/tree/develop
-[1.0.0]: https://github.com/idealista/nexus-role/tree/1.0.0
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
+
+## [Unreleased]
+
+
+## [1.1.0] (2018-06-07)
+### Added
+- *[#2](https://github.com/idealista/nexus-role/issues/2) Add ldap user group* @jmonterrubio
```

Comment:
Probably this change is not released yet

Reply:
I'm getting a jump :)